### PR TITLE
<CloseButton/> - migrate to sections documentation

### PR DESCRIPTION
**Pull request by mykas**
This PR migrates to sections based docs.

### CODE REVIEW

**Comment by mykas** [comment]
@milkyfruit please review and assist on these points:
- component description
- descriptions for examples

File: src/CloseButton/docs/index.story.js [hunk 1]
```diff
@@ -27,5 +57,65 @@ export default {
...
+        <Layout gap={0}>
+          <CloseButton />
+        </Layout>
+      ),
+      sourceUrl:
+        'https://github.com/wix/wix-style-react/tree/master/src/CloseButton/CloseButton.js',
+      issueUrl: 'https://github.com/wix/wix-style-react/issues/new',
+    }),
+    tabs({
+      tabs: [
```
Comment (argshook) [inline src/CloseButton/docs/index.story.js]: you can now do `tabs([ tab() ])` no need for object

File: src/CloseButton/docs/index.story.js [hunk 2]
```diff
@@ -27,5 +57,65 @@ export default {
...
+        tab({
+          title: 'Description',
+          sections: [
+            importExample({
+              source: "import IconButton from 'wix-style-react/IconButton';",
```
Comment (argshook) [inline src/CloseButton/docs/index.story.js]: `importExample("import IconButton from 'wix-style-react/IconButton';")`

File: src/CloseButton/docs/index.story.js [hunk 3]
```diff
@@ -27,5 +57,65 @@ export default {
...
+        }),
+
+        tab({
+          title: 'API',
+          sections: [api()],
+        }),
+
+        tab({
+          title: 'Testkit',
+          sections: [description({ text: testkit })],
```
Comment (argshook) [inline src/CloseButton/docs/index.story.js]:
```suggestion
          sections: [description(testkit)],
```

File: src/CloseButton/docs/index.story.js [hunk 4]
```diff
@@ -1,9 +1,39 @@
...
+
+import * as examples from './examples';
+
+const liveCode = config =>
+  baseLiveCode({
+    previewProps: {
+      style: { backgroundColor: '#f0f4f7' },
+    },
+    compact: true,
+    components: { ...baseScope },
```
Comment (argshook) [inline src/CloseButton/docs/index.story.js]:
```suggestion
    components: baseScope,
```

File: src/CloseButton/docs/index.story.js (hunk 2 above)
Comment (mykas) [reply src/CloseButton/docs/index.story.js]: 😱

**Pull request merged**